lua/nvim-treesitter/parsers.lua

```diff
@@ -303,6 +303,13 @@ list.vue = {
   }
 }
 
+list.jsonc = {
+  install_info = {
+    url = "https://gitlab.com/WhyNotHugo/tree-sitter-jsonc.git",
+    files = { "src/parser.c" },
```

Comment:
You should add yourself as maintainer of the jsonc with `maintainers = {"@foo"}` so it's listed as maintained in the readme :)

Reply:
Also `  readme_name = "JSON with comments",` would be helpful to explain what `jsonc` is.

Reply:
Done here: https://github.com/nvim-treesitter/nvim-treesitter/pull/957